neurovault/apps/statmaps/templates/statmaps/search.html

```diff
@@ -67,11 +67,15 @@
 		    var s = document.getElementsByTagName('script')[0];
 		    s.parentNode.insertBefore(gcse, s);
 		  })();
+
 		</script>
-		<gcse:search linkTarget="_self"></gcse:search>
+		<!-- <gcse:search linkTarget="_self"></gcse:search> -->
+    <gcse:searchbox-only resultsUrl="http://192.168.99.103/search" queryParameterName="as_q"></gcse:searchbox-only>
```

Comment:
Yeah. It's not required. I used for reference. Changes done. (URL should be neurovault.org/search).